lib/handlers/formatting.js

```diff
@@ -47,16 +47,21 @@ export class FormattingHandler {
         }
         const versionArg = this.ceProps(`formatter.${formatter}.version`, '--version');
         const versionRe = this.ceProps(`formatter.${formatter}.versionRe`, '.*');
+        const hasExplicitVersion = this.ceProps(`formatter.${formatter}.explicitVersion`, '') !== '';
         try {
             const result = await exec.execute(exe, [versionArg], {});
             const match = result.stdout.match(versionRe);
             const formatterClass = getFormatterTypeByKey(type);
             const styleList = this.ceProps(`formatter.${formatter}.styles`);
             const styles = styleList === '' ? [] : styleList.split(':');
+            // If there is an explicit version, grab it. Otherwise try to filter the output
+            const version = hasExplicitVersion
+                ? this.ceProps(`formatter.${formatter}.explicitVersion`)
+                : (match ? match[0] : result.stdout);
             this.formatters[formatter] = new formatterClass({
-                exe: exe,
-                version: match ? match[0] : result.stdout,
```

Comment:
Gofmt does not output its version number anywhere, so we needed a way to override the version tag. the new `explicitVersion` properties key allows setting a custom version.